Bottom: ab82ac2be62b27234324b374b250afe4f2656272
Top:    1373cf4206c8556cf6ba7b61e00018738e38f9f1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 15:13:17 +0000

More work in progress


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 806126b..9ee2c34 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2956,8 +2956,6 @@ M_Set_Parentage (POLYAREA *poly, POLYPARENTAGE parentage)
 
 }
 
-static void clear_marks (POLYAREA * p);
-
 /* just like poly_Boolean but frees the input polys */
 int
 poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
